tfe/resource_tfe_policy_set.go

```diff
@@ -43,10 +45,50 @@ func resourceTFEPolicySet() *schema.Resource {
 				ConflictsWith: []string{"workspace_external_ids"},
 			},
 
+			"vcs_repo": {
+				Type:          schema.TypeList,
```

Comment:
`TypeList` tripped me up but judging from the other resources, that's right!

Reply:
I switched it from `TypeSet` to `TypeList` a few months ago. I would need to search Asana (which I can't anymore :wink:) to recall why we needed to make that change, but it do remember was in order to fix or improve something.